bin/get-amy.py

```diff
@@ -18,15 +18,22 @@ def handle_args():
                         help='Base URL of AMY API used for gathering information.')
     parser.add_argument('-o', '--output', default=sys.stdout,
                         help='Output file.  By default: stdout.')
-    parser.add_argument('-t', '--tag', action='append', choices=['SWC', 'DC'],
-                        help='Filter events by this tag.  '
-                             'You can use it multiple times.')
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('--tags-all', action='store', default=None,
+                       help='Events should have any number of these tags, but '
+                            'at least one. Mutually exclusive with --tags-all.'
+                            'Separate tags with a comma, like this: '
+                            '--tags-any=SWC,DC')
+    group.add_argument('--tags-any', action='store', default=None,
+                       help='Events should have all of these tags. Mutually '
+                            'exclusive with --tags-any. Separate tags with '
+                            'a comma, like this: --tags-all=SWC,DC')
 
     args = vars(parser.parse_args())
     return args
 
 
-def main(amy_url, output_file, tags):
+def main(amy_url, output_file, tags, all_=True):
```

Comment:
Hm. What about tags_all and tags_any as arguments, both defaulting to [], and then a test inside of the form:

if (tags_all and tags_any): error
elif tags_all: handle that case
elif tags_any: handle that case
else: no tags